Task2/config/constants.js

```diff
@@ -0,0 +1,17 @@
+const NOTFOUND = '404';
```

Comment:
This could be added as a property under `ErrorStatus` and that whole object can be exported.

Reply:
Done